### PR TITLE
add tests for ecube data gen

### PR DESCRIPTION
**Pull request by leoscholl**
i added more comprehensive tests for the ecube data generation / loading since i wasn't getting the same thing back when i loaded a test signal.

but there doesn't seem to be anything wrong with it.. i'm stumped

### CODE REVIEW

**Comment by leoscholl** [comment]
ok i figured it out, the data just has to have small enough values otherwise it gets clipped when you divide by voltsperbit.

File: aopy/utils.py [hunk 1]
```diff
@@ -73,21 +79,16 @@ def save_test_signal_ecube(data, save_dir, voltsperbit):
...
     filepath = os.path.join(save_dir, filename)
     with open(filepath, 'wb') as f:
-        for _ in range(8):
-            f.write(np.byte(1)) # 8 byte timestamp
-        for t in range(intdata.shape[0]):
-            for ch in range(intdata.shape[1]):
-                f.write(np.byte(intdata[t,ch]))
-                f.write(np.byte(intdata[t,ch] >> 8))
+        f.write(timestamp)
+        f.write(flatdata.tobytes())
```
Comment (zaicheek) [inline aopy/utils.py:91]: i did not know about this function, thank you.

**Review by zaicheek — APPROVED** [review]
docstrings in order, learned a new function for how to wrap data into bytes

**Comment by leoscholl** [comment]
i still can't figure out why the images are slightly different in the test, causing the failure. it also seems to depend on who is running the test because it passes on my machine but not here.

**Comment by leoscholl** [comment]
changes made elsewhere

**Pull request closed without merging**